pkg/cmd/templatefuncs_test.go

```diff
@@ -277,3 +277,41 @@ func TestQuoteListTemplateFunc(t *testing.T) {
 		`"true"`,
 	}, actual)
 }
+
+func TestStrEqFoldTemplateFunc(t *testing.T) {
```

Comment:
As a high-level comment on this test, it's great that this PR includes a test. The test cases here are really testing Go's `strings.EqualFold` function. I think it's reasonable to assume that this works correctly as it's part of Go's standard library, which is already well tested, and so the extra test cases beyond the first one don't add a huge amount of value. Instead, I would suggest a high-level test like the [tests in `templatefuncs.txtar`](https://github.com/twpayne/chezmoi/blob/v2.24.0/pkg/cmd/testdata/scripts/templatefuncs.txtar) which test that the function is plumbed into chezmoi correctly.

Reply:
Correct, thanks!